website/docs/gs/quick_start.md

```diff
@@ -15,12 +15,12 @@ demo data on a managed Cromwell server.
 ### Setup Environment
 
 - A running instance of a Cromwell server. While you can 
-  [host your own Cromwell server](https://cromwell.readthedocs.io/en/stable/tutorials/ServerMode/),
-  (or [these steps](cromwell#setup-cromwell)) due to the required domain expertise, we recommend using a managed 
-  Cromwell instance for the purpose of _Quick Start_.
+  [host your own Cromwell server](https://cromwell.readthedocs.io/en/stable/tutorials/ServerMode/), 
+  we recommend using a managed Cromwell instance for the purpose of _Quick Start_ 
+  (i.e., a server managed by your institute).
```

Comment:
Not sure if this is helpful to say - if a group has access to a cromwell server then they would know that, but if they don't, this may just be confusing.

Reply:
Removed.